packages/core/data-transfer/src/strapi/providers/local-destination/__tests__/assets.test.ts

```diff
@@ -45,6 +45,7 @@ const strapiFactory = getStrapiFactory({
 
 describe('Local Strapi Destination Provider - Get Assets Stream', () => {
   test('Throws an error if the Strapi instance is not provided', async () => {
+    /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
     /* @ts-ignore: disable-next-line */
```

Comment:
Maybe it's better to use something the linter allows 🤷 And maybe favor // for single line comments
```suggestion
		// @ts-expect-error ignore missing properties
```